x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/resource_findings/resource_findings_container.tsx

```diff
@@ -98,10 +99,15 @@ export const ResourceFindings = ({ dataView }: FindingsBaseProps) => {
         <PageTitleText
           title={
             <div style={{ padding: euiTheme.size.s }}>
```

Comment:
can those paddings be removed? are they intended to keep space from the `back to grouped findings` button? not sure they are needed, rules page doesn't use padding for that

Reply:
not sure. this page has a different title than the other 2 pages so it makes sense it'll have some custom stuff 

feel free to remove them if you find it to be not needed